core/client/fs/src/main/java/alluxio/client/file/DoraCacheFileSystem.java

```diff
@@ -191,9 +191,9 @@ public void rename(AlluxioURI src, AlluxioURI dst, RenamePOptions options)
   public void iterateStatus(AlluxioURI path, ListStatusPOptions options,
                             Consumer<? super URIStatus> action)
       throws FileDoesNotExistException, IOException, AlluxioException {
-    AlluxioURI ufsFullPath = convertAlluxioPathToUFSPath(path);
-
-    mDelegatedFileSystem.iterateStatus(ufsFullPath, options, action);
+//    AlluxioURI ufsFullPath = convertAlluxioPathToUFSPath(path);
+//    mDelegatedFileSystem.iterateStatus(ufsFullPath, options, action);
+    this.listStatus(path, options).forEach(action);
```

Comment:
Yes, this api `iterateStatus` is used by bin/alluxio, so this will force it to use the listStatus() api.

I like this.